packages/inter-protocol/src/proposals/addAssetToVault.js

```diff
@@ -125,12 +125,13 @@ export const publishInterchainAssetFromBank = async (
       assetKind: AssetKind.NAT,
     },
   };
-  const { creatorFacet: mint, publicFacet: issuer } = E.get(
+  const { creatorFacet: mint, publicFacet: issuerP } = E.get(
     E(zoe).startInstance(mintHolder, {}, terms),
   );
 
-  const [brand, runBrand] = await Promise.all([
-    E(issuer).getBrand(),
+  const [issuer, brand, runBrand] = await Promise.all([
```

Comment:
The motivation for awaiting it is the call to `E(E(agoricNamesAdmin).lookupAdmin('issuer')).update(keyword, issuer)`. What goes in `agoricNames` should be a presence rather than a promise so that it can be compared for identity.

Reply:
... and so that the board marshaller can serialize it